crates/uv/src/commands/pip/operations.rs

```diff
@@ -566,14 +566,71 @@ pub(crate) async fn install(
     Ok(changelog)
 }
 
+/// Display a message about the target environment for the operation.
+pub(crate) fn report_target_interpreter(
+    python: &PythonInstallation,
+    dimmed: bool,
+    printer: Printer,
+) -> Result<(), Error> {
+    let managed = python.source().is_managed();
+    let implementation = python.implementation();
+    let interpreter = python.interpreter();
+
+    if dimmed {
```

Comment:
Pretty clunky but like... in `uv venv`, we show this at full color, with cyan paths. In `uv pip install`, we show it dimmed. (This is largely preserving existing behavior, open to changing one of the two.)